core/liblwm2m.c

```diff
@@ -68,7 +68,7 @@ lwm2m_context_t * lwm2m_init(void * userData)
     {
         memset(contextP, 0, sizeof(lwm2m_context_t));
         contextP->userData = userData;
-        srand((int)lwm2m_gettime());
+        srand((int)lwm2m_seed());
```

Comment:
The type cast to `int` is not needed anymore, since `lwm2m_seed()` already returns an `int`.

Reply:
Fixed